assets/docs/neso613/ASR_TFLite/pre_trained_models/English/1.md

```diff
@@ -0,0 +1,41 @@
+# Lite neso613/ASR_TFLite/pre_trained_models/English/1
+
+TF Lite quantized version of the Conformer to generate text from audio.
+
+<!-- parent-model: neso613/ASR_TFLite/pre_trained_models/1 -->
+<!-- asset-path: legacy -->
+
+ASR TFLite model is 19.1 MB in size[link.](https://drive.google.com/file/d/1jElSz35ivHsW7ww5X85rghEf0PVDlaMZ/view?usp=sharing)
+
+## About Conformer
```

Comment:
It could be helpful to provide some information about the inputs and outputs. It's unclear to me right now what this model does.

Reply:
The TFLite model takes english audio as input and gives english text as output.
I will update this in readme also.